libs/langchain/langchain/chat_models/konko.py

```diff
@@ -122,23 +122,23 @@ def _default_params(self) -> Dict[str, Any]:
 
     @staticmethod
     def get_available_models(
-        konko_api_key: Optional[str] = None,
-        openai_api_key: Optional[str] = None,
+        konko_api_key: Optional[SecretStr] = None,
```

Comment:
This is a publicly exposed static method -- let's avoid making a breaking change here in case users are relying on using this static method directly. Instead could you keep the api keys here `str` (there's no need to mask them here since they won't get printed). If we think they'll be used in conjunction with the existing keys on set as attribtues on the model, then we can make these types: Union[str, SecretStr, None]  and support both `str` and `SecretStr` as inputs